Create a database based on a diagram

enoqueJonas opened the pull request:
# Create a database based on a diagram

## This PR was raised to:
- Diagram a data model clearly and explain what it's doing.

## What does this PR do:
- Create a file named schema_based_on_diagram.sql to implement the database from the diagram.
- Create join tables for many-to-many relationships.
- Add the FK indexes.

## Improvements to be made:
- None

enoqueJonas committed:
UPDATE schema file name

enoqueJonas committed:
UPDATE schema by adding new tables

SimonGideon committed:
invoice item tabls

algerina reviewed (changes requested):
Hi @enoqueJonas ,

Good job so far!  But still some minor issues to complete.

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your implemented all the requested tables using the provided Diagram:heavy_check_mark:



## STATUS: REQUIRED CHANGES 


<img src="https://media3.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif"/>

 ##required changes:

- [x] Kindly Update your Readme file by deleting all unnecessary sections.
- [x] The MIT license link is not working, kindly use a working link to your license.

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

enoqueJonas committed:
UPDATE README.md

enoqueJonas committed:
ADD MIT.md

enoqueJonas committed:
Merge branches 'create-schema' and 'create-schema' of github.com:enoqueJonas/schema_based_on_diagram into create-schema

danmainah commented on README.md:
```diff
@@ -1,9 +1,3 @@
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
 ## Getting Started
```
- [ ] @team as previously requested kindly remove this section since you already have getting started section you could choose to add info from this section missing in the other section.

danmainah commented on README.md:
```diff
@@ -14,81 +8,41 @@ This repository includes files with plain SQL that can be used to recreate a dat
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Schema Based On Diagram <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```
- [ ] @team  kindly remove this line, its suppose to guide you when writing the readme but shouldn't be retained in the readme.

danmainah commented on README.md:
```diff
@@ -14,81 +8,41 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```
- [ ]  @team kindly remove this line, its suppose to guide you when writing the readme but shouldn't be retained in the readme.

danmainah commented on README.md:
```diff
@@ -102,9 +56,7 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```
- [ ]  @team kindly remove this line, its suppose to guide you when writing the readme but shouldn't be retained in the readme.

danmainah reviewed (changes requested):
Hi @team, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

enoqueJonas commented:
Hello @danmainah. I already mad those changes. Please check my repo again.

danmainah commented:
> Hello @danmainah. I already mad those changes. Please check my repo again.

I have already checked it here is a screenshot highlighting  one of them, you can see the branch name at the top and how your readme looks
![image](https://user-images.githubusercontent.com/61408860/217357560-7e4441cc-2c56-4e53-811c-3a7e3229370f.png)

enoqueJonas commented:
From the screenshot, I cannot see where's the problem. The screenshot shows that I made the changes.
If you check my code you'll see that I made the changes. I deleted the unnecessary text and added new text.
What you highlighted shows that I made the changes.
Red shows that I deleted something, and green shows that I added something.
Please check again @danmainah.

enoqueJonas commented:
[Here](https://github.com/enoqueJonas/schema_based_on_diagram/pull/1#:~:text=this%20form.-,README.md,to%20add%20info%20from%20this%20section%20missing%20in%20the%20other%20section.,-Reply...) is an example. I already deleted this.

medaminedev66 commented:
Hello @enoqueJonas 👋,

_This is an additional review as a result of the second opinion request._

@danmainah requested reasonable changes. Your README is the entry point of your users and having it well-written gives a good first impression and maximizes the potential of your product to your users.

Please customize your Readme and remove the guides as mentioned by @danmainah. Those lines are only there to help you when writing the readme; they shouldn't be included.

The following lines should be removed:
> Describe your project in 1 or 2 sentences.
> Describe the tech stack and include only the relevant sections that apply to your project.
> Describe between 1-3 key features of the application.
> Add a link to your deployed project.

Keep rocking 🚀!

enoqueJonas committed:
UPDATE README.md

enoqueJonas committed:
UPDATE README.md

ShadyShawkat reviewed (approved):
## Hi @enoqueJonas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

![Image](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
